app/src/main/java/com/cornellappdev/android/eatery/CampusMenuActivity.java

```diff
@@ -211,8 +213,44 @@ else if (mCafeData instanceof DiningHallModel) {
             }
         }
 
-        // Set up wait times feature.
-        mWaitTimesFragment = new WaitTimesFragment();
+        this.setupWaitTimes();
+    }
+
+    // Set up wait times feature.
+    private void setupWaitTimes() {
```

Comment:
wait, i just realized -- this method doesn't not belong in an activity class. did you create a presenter for the `CampusMenuActivity`?

Reply:
Presenter created in last push